src/views/SeedlotCreatedFeedback/styles.scss

```diff
@@ -0,0 +1,59 @@
+@use '../../theme/variables.scss' as vars;
+@use '@carbon/type';
+
+.seedlot-created-feedback-page{
+  display: flex;
+  align-items: center;
+  height: 100vh;
+  margin-right: 8.06rem;
+}
+
+.scf-pic-container{
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.scf-pictogram{
+  width: 23.125rem;
+  height: 22.375rem;
+  color: var(--#{vars.$bcgov-prefix}-focus);
```

Comment:
The color defined on the figma file is the primary button color (Light Theme/Button/$button-primary), adjust it here to reflect correctly the design.